rpc/handler.go

```diff
@@ -388,7 +406,23 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage, stream *json
 		}
 		return nil
 	case msg.isCall():
+		if h.rpcSlowLogThreshold > 0 && h.isRpcMethodNeedsCheck(msg.Method) {
+			slowTimer := time.AfterFunc(time.Duration(h.rpcSlowLogThreshold)*time.Millisecond, func() {
+				h.logger.Warn("[rpc.slow] - running", "method", msg.Method)
```

Comment:
is it possible to print request ID and request body here? (and print only ID at `finish`)